static/templates/compose_control_buttons.hbs

```diff
@@ -1,15 +1,18 @@
-<input type="file" id="file_input" class="notvisible pull-left" multiple />
+{{! This file assumes message_id is only preset in message_edit_form and message_id cannot be 0.}}
+<input type="file" {{#if message_id }}id="message_edit_file_input_{{message_id}}"{{ else }}id="message_edit_file_input_{{message_id}}"{{/if}} class="notvisible pull-left" multiple />
```

Comment:
This seems to od the same thing in both if/else clauses, which seems like a bug.